### PR TITLE
ITest: fix get function of state parameter

### PR DESCRIPTION
**Pull request by simonzihlmann**
This PR fixes an error in the get function of the state parameter and changes default value of synchronous_delay parameter and delete trailing white spaces.

Previously while getting  the state parameter of a channel (15 for example) by `dac.channels[15].state()`, the following error was raised:

```
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
~\Miniconda3\lib\site-packages\qcodes\instrument\parameter.py in _from_raw_value_to_value(self, raw_value)
    563                 try:
--> 564                     value = self.inverse_val_mapping[int(value)]
    565                 except (ValueError, KeyError):

KeyError: 0

During handling of the above exception, another exception occurred:

KeyError                                  Traceback (most recent call last)
<ipython-input-6-a87cfc1210c9> in <module>
----> 1 dac.channels[15].state()

~\Miniconda3\lib\site-packages\qcodes\instrument\parameter.py in __call__(self, *args, **kwargs)
    397         if len(args) == 0:
    398             if self.gettable:
--> 399                 return self.get()
    400             else:
    401                 raise NotImplementedError('no get cmd found in' +

~\Miniconda3\lib\site-packages\qcodes\instrument\parameter.py in get_wrapper(*args, **kwargs)
    590             except Exception as e:
    591                 e.args = e.args + (f'getting {self}',)
--> 592                 raise e
    593 
    594         return get_wrapper

~\Miniconda3\lib\site-packages\qcodes\instrument\parameter.py in get_wrapper(*args, **kwargs)
    579                 raw_value = get_function(*args, **kwargs)
    580 
--> 581                 value = self._from_raw_value_to_value(raw_value)
    582 
    583                 if self._validate_on_get:

~\Miniconda3\lib\site-packages\qcodes\instrument\parameter.py in _from_raw_value_to_value(self, raw_value)
    564                     value = self.inverse_val_mapping[int(value)]
    565                 except (ValueError, KeyError):
--> 566                     raise KeyError(f"'{value}' not in val_mapping")
    567 
    568         return value

KeyError: ("'False' not in val_mapping", 'getting dac_chan16_state')
```

The PR fixes this.

### CODE REVIEW

File: qcodes_contrib_drivers/drivers/Bilt/ITest.py [hunk 1]
```diff
@@ -118,7 +119,7 @@ def __init__(self, parent: Instrument,
                            get_parser=float,
                            set_cmd=None,
                            vals=vals.Numbers(1e-3, 10),
-                           initial_value=1)
+                           initial_value=1e-3)
```
Comment (jenshnielsen) [inline qcodes_contrib_drivers/drivers/Bilt/ITest.py:122]: Is this change intentional ? Does not seem related

Comment (jenshnielsen) [reply qcodes_contrib_drivers/drivers/Bilt/ITest.py:122]: Sorry some how missed this

Comment (simonzihlmann) [reply qcodes_contrib_drivers/drivers/Bilt/ITest.py:122]: no problem, it is indeed not related to the fix but nice to change the default value.

**Review by FarBo — APPROVED** [review]
